cmd/agent/app/run.go

```diff
@@ -277,6 +277,14 @@ func StartAgent() error {
 		log.Info("logs-agent disabled")
 	}
 
+	// Try to read in System Probe config if process agent is enabled
+	if config.Datadog.GetBool("process_config.enabled") {
```

Comment:
Don't you want to check if the system probe is enabled?

 I can have the `connections` check enabled without `process_config.enabled` being true (i.e. container + process collection is disabled)

Reply:
Ah, this is my mistake, I was unaware that processes can be disabled, but system probe still be enabled. 

My goal here was to limit the number of places we would throw a warning log line about the system probe configuration file not being available. I'm going to look into other options.